nativelink-service/tests/ac_server_test.rs

```diff
@@ -48,29 +49,46 @@ async fn insert_into_store<T: Message>(
     let data_len = store_data.len();
     let digest = DigestInfo::try_new(hash, action_size)?;
     store.update_oneshot(digest, store_data.freeze()).await?;
+
     Ok(data_len.try_into().unwrap())
 }
 
 async fn make_store_manager() -> Result<Arc<StoreManager>, Error> {
     let store_manager = Arc::new(StoreManager::new());
-    store_manager.add_store(
+    make_and_add_store_to_manager(
         "main_cas",
-        store_factory(
-            &StoreSpec::memory(MemorySpec::default()),
-            &store_manager,
-            None,
-        )
-        .await?,
-    );
-    store_manager.add_store(
+        &StoreSpec::memory(MemorySpec::default()),
+        &store_manager,
+        None,
+    )
+    .await?;
+
+    let current_dir = env::current_dir().expect("Failed to get current directory");
+
+    let default_filesystem_spec = FilesystemSpec::default();
+
+    make_and_add_store_to_manager(
         "main_ac",
-        store_factory(
-            &StoreSpec::memory(MemorySpec::default()),
-            &store_manager,
-            None,
-        )
-        .await?,
-    );
+        &StoreSpec::filesystem(FilesystemSpec {
+            content_path: current_dir
```

Comment:
This is too complicated. Would be simpler to keep `MemorySpec` here. I know it isn't pure but we don't have consistent way to test `FilesystemStore`. We need one.